### ISSUE TITLE
FireGestures using Web Extensions? 

### ISSUE THREAD

**Issue by Lurtzae** [gomita/firegestures#149]
Is there any way this extension will survive the transition to Web Extensions? Are there plans to build Fire Gestures as a Web Extension?

**Comment by lgommans** [comment]
#168 sounds like a duplicate. Though this one is older and has more +1s, I'd close this one in favor of the other one since that one has more comments.

**Comment by Grossdm** [comment]
I agree with @lgommans .  Please consider closing this issue as a duplicate of [#168](https://github.com/gomita/firegestures/issues/168).
